sqlalchemy_jsonapi/declarative/serializer.py

```diff
@@ -0,0 +1,82 @@
+"""A serializer for serializing SQLAlchemy models to JSON API spec."""
+
+
+class JSONAPISerializer(object):
+    """A JSON API serializer that serializes SQLAlchemy models."""
+
+    def serialize(self, resources):
+        """Serialize resource(s) according to JSON API spec.
+
+        Args:
+            resources: SQLAlchemy model or SQLAlchemy query
+        """
+        if 'id' not in self.fields:
+            raise AttributeError
+
+        serialized_object = {
+            'included': [],
+            'jsonapi': {
+                'version': '1.0'
+            },
+            'meta': {
+                'sqlalchemy_jsonapi_version': '4.0.9'
```

Comment:
Is there a way we can reference the version such as __version__? That way we don't have to keep refactoring tests in future release?